request-manager/grapher/grapher.go

```diff
@@ -342,8 +342,29 @@ func (o *Grapher) buildComponent(name string, nodeDefs map[string]*NodeSpec, nod
 				}
 
 				// Insert all components between the start and end vertices.
+				// Place in parallel if possible, otherwise serialize.
```

Comment:
instead of saying `if possible`, explain how they're placed in parallel / serialized according to the `Parallel` field on the node (if one exists)